docs/kh2/dictionary/icons.md

```diff
@@ -0,0 +1,66 @@
+# [Kingdom Hearts II](index.md) - Icons
+
```

Comment:
Could I request if you merge the list of icons with the one from [this](https://openkh.dev/tool/GUI.Kh2TextEditor/) page? About the images, it would be preferable to move the pictures inside the `/docs/kh2` folder and redirect the url from `/tool/GUI.Kh2TextEditor/` to the new location.

Reply:
Moved the icons to /kh2/images/icons.